scripts/maintenance/fix_remaining_py38_types.py

```
@@ -0,0 +1,281 @@
+        new_imports = ', '.join(sorted(imports_to_add))
+        if existing_imports:
+            # Add to existing import
+            content = re.sub(
+                r'from typing import ([^\\n]+)',
+                f'from typing import {existing_imports}, {new_imports}',
+                content
+            )
+        else:
+            # Replace empty import
+            content = re.sub(
+                r'from typing import', 
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)